### ISSUE TITLE
Not working onChange property in select input type

### ISSUE THREAD

**Issue by felipecock** [fateh999/react-native-paper-form-builder#31]
The _'select'_ type input does not have a property to trigger an event handler when it changes.
I have tried with _onChange_ into _textInputProps_, but it won't trigger because this is actually a Menu item with a _onDismiss_ void property that has not been linked to the _InputSelect_ props.

```
<FormBuilder
  control={control}
  setFocus={setFocus}
  formConfigArray={[
    {
      name: 'favoriteFruit',
      type: 'select',
      textInputProps: {
        label: 'What is you favorite fruit?',
        onChange: fruit =>
          console.log('Your favorite fruit is: ', fruit),
      },
      options: [
        {
          value: 'mango',
          label: 'Mango',
        },
        {
          value: 'banana',
          label: 'Banana',
        },
        {
          value: 'strawberry',
          label: 'Strawberry',
        },
        {
          value: 'pineapple',
          label: 'Pineapple',
        },
        {value: 'other', label: 'Other'},
      ],
      rules: {
        required: {
          value: true,
          message: 'This field is required.',
        },
      },
    },
  ]}
/>
```

**Comment by fateh999** [comment]
Ok i will add a callback, for this use case

**Comment by felipecock** [comment]
#32

**Comment by felipecock** [comment]
> Ok i will add a callback, for this use case

Excelent, thank you Fateh.

**Comment by felipecock** [comment]
Hello Fateh, did you have time to take a look at my PR #32 to add this feature?
I'm really needing this urgent to fix a bug in my application. Thank you very much and have a nice weekend.

**Comment by fateh999** [comment]
Thanks for the PR, i have merged it in the latest release v2.0.5
